care/facility/api/serializers/patient_consultation.py

```diff
@@ -192,6 +196,34 @@ def validate_bed_number(self, bed_number):
             bed_number = None
         return bed_number
 
+    def get_mews_field(self, consultation):
+        current_time = localtime(now())
+        past_30_minutes = current_time - timedelta(minutes=30)
+        recentDailyArray = DailyRound.objects.filter(rounds_type = DailyRound.RoundsType.NORMAL.value ,consultation = consultation ,modified_date__gte=past_30_minutes, modified_date__lte=current_time).order_by("-modified_date")
```

Comment:
for the filtering five data, these data will not be available in non-normal round types in the time range thats why I'm filtering only normal types because there are only two types of round types

Reply:
if there are 9 logs created 4 normal, and 5 critical, just hypothetical case recently in 30 min then if i fetch all logs, then I need to traverse in all the 9 files, that would increase time and it couldn't be more optimal, I think...?

Reply:
Are you sure this PR solves this issue? Virtual Nursing Assistant creates Automated rounds and not normal rounds

<img width="943" alt="image" src="https://github.com/coronasafe/care/assets/25143503/6e86ee54-8c6a-40f1-9668-32571ff54f8b">

Reply:
Yeah it's that

Reply:
Okay I'll not filter I'll erase that filtering parameters of round type

Reply:
done✅